tests/conftest.py

```diff
@@ -494,6 +493,20 @@ def minicircle_small_objects_removed(minicircle_minimum_grain_size: np.array) ->
     return minicircle_minimum_grain_size
 
 
+@pytest.fixture
+def minicircle_area_thresholding(minicircle_grain_labelled_all: np.array, grains_config: dict) -> Grains:
+    """Small objects removed."""
+    minicircle_grain_labelled_all.directions["upper"][
+        "removed_small_objects"
+    ] = minicircle_grain_labelled_all.area_thresholding(
+        minicircle_grain_labelled_all.directions["upper"]["labelled_regions_01"],
+        grains_config["area_absolute_threshold_upper"],
+        grains_config["area_absolute_threshold_lower"]
+    )
+    # for some reason this is returning an image with most molecules gone
```

Comment:
Is that because they are outside of the area threshold?

Reply:
I don't believe so as these are the current values I use when normally running TS to gather the expected amount of grains.

Using some print statements I find the following:
Normal TS run:
Labelled grains found: 126
Initial grains found: 126
Final grains found: 21
Thresh: (500, 800)

Test TS run:
Labelled grains found: 41
Initial grains found: 41
Final grains found: 3
Thresh: (500, 800)

So I'm thinking I might be using the wrong image?